### PR TITLE
Auto install shell completions on CLI install

### PR DESCRIPTION
**Pull request by Piccirello**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -136,6 +145,11 @@ nfpms:
     formats:
       - deb
       - rpm
+    contents:
+      - src: ./completions/doppler.bash
+        dst: /etc/bash_completion.d/doppler
```
Comment (ryan-blunden) [inline .goreleaser.yml:150]: I honestly don't know much about bash-completion, or the install/activation flow, but do we need to take into account env vars such as `$BASH_COMPLETION_DIR`, or if on macOS looking for directory paths such as `/usr/local/etc/bash_completion.d` that don't require sudo/root access to install?

Comment (Piccirello) [reply .goreleaser.yml:150]: I don't know very much about bash-completion either, I wasn't even aware of its custom environment variables. I'm not sure how we would accomplish that with our current tooling to be honest.

This section is only for the deb and rpm repositories, so it'll be used exclusively on linux. It's worth noting that the binary is installed to `/usr/bin` and thus already requires sudo access.

**Review by nmanoogian — APPROVED** [review]

File: .goreleaser.yml (hunk 1 above)
Comment (rgmvisser) [reply .goreleaser.yml:150]: Can we take a look at maybe another good open source CLI package that has autocompletion? Maybe we can get some best practices from there?

Comment (Piccirello) [reply .goreleaser.yml:150]: This change was unabashedly borrowed from goreleaser's own [config](https://github.com/goreleaser/goreleaser/blob/master/.goreleaser.yml). I neglected to document this anywhere other than the linear task (which isn't linked here).

**Review by rgmvisser — APPROVED** [review]

**Pull request merged**